src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -528,16 +541,16 @@ public function viewAction(int $orderId, Request $request): Response
 
         $metatitle = sprintf(
             '%s %s %s',
-            $this->trans('Orders', 'Admin.Orderscustomers.Feature'),
-            $this->getConfiguration()->get('PS_NAVIGATION_PIPE', '>'),
+            $this->trans('Orders', [], 'Admin.Orderscustomers.Feature'),
+            $this->getConfiguration()->get('PS_NAVIGATION_PIPE'),
```

Comment:
```suggestion
            $this->getConfiguration()->get('PS_NAVIGATION_PIPE') ?? '>',
```

Just to keep the default value behaviour, it's quite annoying that the interface doesn't include the second parameter, maybe we should add it 🤔 Not sure it would be a breaking change if it's optional